### PR TITLE
Update NLS Message

### LINKED ISSUE #8708: MP: Update NLS messages after review in PR #8691
https://github.com/OpenLiberty/open-liberty/pull/8691#discussion_r316207643

### PR DESCRIPTION
**Pull request by tevans78**
#build 
fixes #8708

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 1]
```diff
@@ -52,8 +52,8 @@ kafka.output.error.signal.CWMRX1004E.useraction=Review the server message.log fi
 
 # The group.id property was not set in the configuration for this channel. Set the {0} property in one of the MicroProfile Config Sources for this application.
 kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set in the configuration for this channel. Set the {0} property in one of the MicroProfile Config Sources for this application.
-kafka.groupid.not.set.CWMRX1005E.explanation=The Kafka Consumer Group ID must be set to allow Reactive Messaging to function properly.
-kafka.groupid.not.set.CWMRX1005E.useraction=Set the property into one of the MicroProfile Config Sources for this application e.g. the microprofile-config.properties file.
+kafka.groupid.not.set.CWMRX1005E.explanation=The Kafka consumer group ID must be set to allow Reactive Messaging to function properly.
+kafka.groupid.not.set.CWMRX1005E.useraction=Set the property in one of the MicroProfile ConfigSource locations for this application, for example, the microprofile-config.properties file.
```
Comment (Rwalls1) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Remove the extra space before "for example".

Comment (tevans78) [reply dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: there is only one space?

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops (hunk 1 above)
Comment (Rwalls1) [reply dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Okay, the formatting appeared to add an extra space to me, but it's fine now

**Pull request merged**